static/js/composebox_typeahead.js

```diff
@@ -244,6 +249,11 @@ function handle_keydown(e) {
         } else if (on_stream || on_topic || on_pm) {
             // We are doing the focusing on keyup to not abort the typeahead.
             if (on_stream) {
+                if (key === "Enter") {
+                    // This is just opening the dropup menu
```

Comment:
TODO: verify that this doesn't also capture closing the dropup menu and explain that here